tools/transfer-client/README.md

```diff
@@ -47,24 +47,18 @@ The expected JSON format of the keys file is
 }
 ```
 
-## Extracting keys from Mobile Wallet
+## Extracting keys from the Browser Wallet
 
-The tool expects account keys to be provided in plain text.
-The mobile wallet only allows keys to be exported with password protected encryption.
-To use keys exported from a wallet, they therefore first need to be decrypted and extracted as follows:
+In the [browser wallet](https://github.com/Concordium/concordium-browser-wallet), go to:
 
-First [create an export](https://developer.concordium.software/en/mainnet/net/mobile-wallet/export-import-mw.html)
-in the app and transfer the export file `concordium-backup.concordiumwallet` to your PC.
+Account Settings -> Export private key -> Export
 
-Then decrypt the export using [utils](https://developer.concordium.software/en/mainnet/net/references/developer-tools.html#decrypt-encrypted-output)
-and extract the key portion using [jq](https://stedolan.github.io/jq/), e.g.:
+This will give you a file. You can extract the key portion using [jq](https://stedolan.github.io/jq/), e.g.:
 
-```shell
-$ utils decrypt --in concordium-backup.concordiumwallet | \
-   jq '.value.identities[0].accounts[0].accountKeys' > sender.keys
-# enter password
+```bash
+jq .value.accountKeys your-account-address-here.export > keys.json
 ```
 
-The keys are now stored in clear text in the file `sender.keys`.
+The keys are now stored in clear text in the file `keys.json`.
 
```

Comment:
All of this is just needless complexity. The Rust SDK can use the keys exported from the browser wallet directly
```
    let sender_keys = WalletAccount::from_json_file(&args.keys_file)?;
```
where `keys_file` is the export file from either the browser wallet, or in the genesis format.

Please simplify this and change the transfer client to use that.